projects/plugins/jetpack/extensions/shared/external-media/editor.scss

```diff
@@ -67,6 +67,110 @@ $grid-size: 8px;
 	}
 }
 
+.jetpack-external-media-browser.is-jetpack-app-media {
+	max-width: 733px;
+
+	.components-modal__content {
+		@media ( min-width: 600px ) {
+			width: 90vw;
+			max-width: 733px;
+			height: 90vh;
+		}
+	}
+	.jetpack-app-icon {
+		width: 69px;
+	}
+
+	.components-modal__content {
+		margin-top: 0;
+		padding: 40px 56px 40px;
+	}
+
+	.jetpack-external-media-browser__media__item {
+		opacity: 1;
+		animation: animate-drop 0.3s ease;
+	}
+	@media only screen and ( min-width: 600px ) {
+		.jetpack-external-media-browser__media__item {
+			padding-top: 96px;
+			width: 112px;
+		}
+	}
+
+	.jetpack-external-media-browser .jetpack-external-media-browser__media__item img {
+		width: 80px;
+  		height: 80px;
+	}
+}
+
+@keyframes animate-drop {
+	0% {
+		opacity: 0;
+		transform: translateY(-100%);
+	}
+	100% {
+		opacity: 1;
+		transform: translateY(0);
+	}
+}
+
+.jetpack-external-media-wrapper__jetpack_app_media-title {
+	font-family: Recoleta, "Noto Serif", Georgia, "Times New Roman", Times, serif;
+	font-size: 24px;
+	font-weight: 400;
+	line-height: 40px;
```

Comment:
Unitless `line-height` is usually safer (40/24=1.67 in that instance). Some browsers could be quirky with this.

Reply:
Great suggestion!